packages/paste-website/src/component-examples/CornerOrnamentExamples.ts

```diff
@@ -0,0 +1,283 @@
+                    100
+                </Badge>
+            </CornerOrnament>
+        </CornerOrnamentContainer>
+        <CornerOrnamentContainer cornerOrnamentType="badge" size="sizeIcon80" position="top_end">
+            <CornerOrnamentBase>
+                <CartIcon color="colorTextIcon" size="sizeIcon80" decorative={false} title="shopping cart" />
+            </CornerOrnamentBase>
+            <CornerOrnament>
+                <ScreenReaderOnly>notification value is</ScreenReaderOnly>
+                <Badge as="span" variant="notification_counter" size="small">
+                    100
+                </Badge>
+            </CornerOrnament>
+        </CornerOrnamentContainer>
+        <CornerOrnamentContainer cornerOrnamentType="badge" size="sizeIcon80" position="top_end">
+            <CornerOrnamentBase>
+                <Avatar
+                    size="sizeIcon80"
+                    name="Tri O'Hara"
+                    src="/images/avatars/avatar4.png"
+                />
+            </CornerOrnamentBase>
+            <CornerOrnament>
+                <ScreenReaderOnly>notification value is</ScreenReaderOnly>
+                <Badge as="span" variant="notification_counter" size="small">
+                    100
+                </Badge>
+            </CornerOrnament>
+        </CornerOrnamentContainer>
+    </Stack>
+  );
+};
+
+render(
+  <BadgeCornerOrnamentExample />
+)`.trim();
+
+export const badgeOverflowCornerOrnamentExample = `
+const BadgeOverflowCornerOrnamentExample = () => {
+  return (
+    <Stack orientation="horizontal" spacing="space70">
+        <CornerOrnamentContainer cornerOrnamentType="badge" size="sizeIcon80">
+            <CornerOrnamentBase>
+                <CartIcon color="colorTextIcon" size="sizeIcon80" decorative={false} title="shopping cart" />
+            </CornerOrnamentBase>
+            <CornerOrnament>
+                <ScreenReaderOnly>notification value is</ScreenReaderOnly>
+                <Badge as="span" variant="notification_counter" size="small">
+                    9+
+                </Badge>
+            </CornerOrnament>
+        </CornerOrnamentContainer>
+        <Button variant="secondary">
+            <ProductAlarmsIcon decorative={true} size="sizeIcon20" />
+            Notifications
+            <Badge as="span" variant="notification_counter" size="small">
+                10
+            </Badge>
+        </Button>
+        <CornerOrnamentContainer cornerOrnamentType="icon" size="sizeIcon80">
+            <CornerOrnamentBase>
+                <ProductAlarmsIcon color="colorTextIcon" size="sizeIcon80" decorative={false} title="shopping cart" />
+            </CornerOrnamentBase>
+            <CornerOrnament>
+                <ScreenReaderOnly>You have a notification</ScreenReaderOnly>
+                <NotificationOrnamentIcon color="colorTextIconNotification" decorative size="sizeIcon40" />
+            </CornerOrnament>
+        </CornerOrnamentContainer>
```

Comment:
todo:
- Use NotificationIcon instead of ProductAlarmsIcon
- Set position to top_end
- Set title to "Notifications"